website/custom.css

```diff
@@ -1,7 +1,15 @@
 code {
   @apply text-sm;
 }
-
+.btn {
```

Comment:
does it belong here?

Reply:
Yes, it's for the Slack plugin docs - it renders the button here: https://cloudquery-nhehfbwxm-cloudquery.vercel.app/docs/plugins/sources/slack/overview
![Screenshot 2022-12-19 at 10 06 00](https://user-images.githubusercontent.com/1121616/208400510-ad93cca4-9c8e-4697-9bc4-bca1e78e9f93.png)